.github/workflows/review.yml

```diff
@@ -0,0 +1,29 @@
+name: Code Review
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    branches: [main]
+
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.event.number || github.head_ref ||
+    github.sha }}-${{ github.workflow }}-${{ github.event_name ==
+    'pull_request_review_comment' && 'pr_comment' || 'pr' }}
+  cancel-in-progress: ${{ github.event_name != 'pull_request_review_comment' }}
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: coderabbitai/ai-pr-reviewer@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GT_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        with:
+          debug: false
+          review_simple_changes: false
+          review_comment_lgtm: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The workflow file seems to be well structured and follows the standard GitHub Actions syntax. However, there are a few points that need attention:

1. The `GITHUB_TOKEN` is being set using a custom secret `GT_TOKEN`. It's recommended to use the provided `secrets.GITHUB_TOKEN` which is automatically created by GitHub for each repository. This token has the necessary permissions to perform most common tasks.

2. There is no newline at the end of the file. While this won't cause any issues with the YAML parsing or the GitHub Actions runner, it's a common convention to end files with a newline. This can help prevent potential issues with certain tools that expect or require it.

Here are the suggested changes:

```diff
-           GITHUB_TOKEN: ${{ secrets.GT_TOKEN }}
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

\ No newline at end of file
+ 
```

Please ensure that the OpenAI API key stored in `secrets.OPENAI_API_KEY` has the correct permissions and is valid. If these keys are not set correctly, the action will fail to run.


<!-- This is an auto-generated comment by OSS CodeRabbit -->